src/components/InstanceMonitoring/_instance-monitoring.scss

```diff
@@ -0,0 +1,51 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/vars';
+
+.infra--instance-card {
+  background-color: $ui-01;
+  width: 100%;
+  height: rem(210px);
+  .infra--instance-metrics {
```

Comment:
Nitpick - Do we want to add a empty lines e.g. above here in a similar manner as other Sass files in Carbon?